src/rsa/bigint.rs

```diff
@@ -218,6 +218,21 @@ pub struct Modulus<M> {
     ring: PhantomData<M>,
 }
 
+impl<M> Modulus<M> {
+    pub fn bit_length(&self) -> bits::BitLength {
+        let bits = unsafe { GFp_BN_num_bits(self.as_ref()) };
+        bits::BitLength::from_usize_bits(bits)
+    }
+
+    pub fn fill_be_bytes(&self, out: &mut [u8])
+                         -> Result<(), error::Unspecified> {
+        bssl::map_result(unsafe {
+            GFp_BN_bn2bin_padded(out.as_mut_ptr(), out.len(),
+                                 self.as_ref())
+        })
+    }
```

Comment:
Let's move this `unsafe` implementation of `fill_be_bytes()` to `Nonnegative`, and then make the `Modulus` and `Elem` implementations be simple wrappers around it.